articles/quickstart/native/android/getting-started.md

```diff
@@ -4,22 +4,22 @@ description: Short Introduction to the Auth0 Android Quickstarts.
 budicon: 715
 ---
 
-This multistep quickstart guide will walk you through managing authentication in your android apps with Auth0.
+This quickstart guide walks you through managing authentication  with Auth0 in your Android apps.
 
 ## Sample Projects
 
-Each tutorial in the series includes a link to its corresponding sample project. You can find all the samples [here](https://github.com/auth0-samples/auth0-android-sample).
+Each tutorial in the series includes a link to a sample project. You can find all the samples [here](https://github.com/auth0-samples/auth0-android-sample).
 
 ## Dependencies
 
-Each tutorial will require you to use the [Auth0.Android](https://github.com/auth0/Auth0.Android) library. This is a toolkit that lets you communicate with many of the basic [Auth0 API](https://auth0.com/docs/api) functions in a neat way.
+Each tutorial uses the [Auth0.Android](https://github.com/auth0/Auth0.Android) library. It is a toolkit that lets you communicate with many of the basic [Auth0 API](https://auth0.com/docs/api) functions in a neat way.
 
 ## Create a Client
 
-If you haven't already done so, create a new client application in your [Auth0 dashboard](${manage_url}/#/applications/${account.clientId}/settings) and choose **Native** for the Type.
+Create a new client application in your [Auth0 dashboard](${manage_url}/#/applications/${account.clientId}/settings). For **Type**, select **Native**.
```

Comment:
Replace the Auth0 Dashboard URL to `${manage_url}/#/clients` as this one needs to be the clients/create clients page